public/js/pages/index.js

```diff
@@ -22,41 +22,32 @@ require(['jquery', 'base/ui', 'base/gallery', 'base/login'],
     });
 
     UI.select('#search-filter', function (val) {
-
       var makes = document.querySelectorAll('.make');
 
-      switch (val) {
-      case 'featured':
+      if (val === 'featured') {
```

Comment:
this needs a stronger rewrite, because as an if/else set, there is a *lot* of code duplication in this block now